crates/bin/src/args.rs

```diff
@@ -75,7 +75,22 @@ pub struct Args {
     pub bin_dir: Option<String>,
 
     /// Override Cargo.toml package manifest pkg-fmt.
```

Comment:
I think you would still need to manually write the help page as doc.
I tried to use `possible_values` for `Arguments::log_level` but it doesn't change the default `--help` output.

Reply:
But in my case, the help message is already looks as I expected

```
        --pkg-fmt <PKG_FMT>
            Override Cargo.toml package manifest pkg-fmt [possible values: Tar, Tbz2, Tgz, Txz,
            Tzstd, Zip, Bin]
 ```
 
 Am I missing something?

Reply:
You are right, perhaps the latest version of clap fixed the issue I met before.

Reply:
The docs says it only displayed in shell completion.  https://docs.rs/clap/latest/clap/struct.PossibleValue.html#method.help
So it would be shown after the user pressing `TAB` on most shell. (if we support shell completions)

Reply:
For shell completions, it seems that [`clap_complete`](https://docs.rs/clap_complete/latest/clap_complete/) is required and the shell completion needs to be installed separately, which makes distributing it even harder consider `cargo-binstall` currently only installs binary.

Reply:
I think just printing `[possible values: Tar, Tbz2, Tgz, Txz, Tzstd, Zip, Bin]` is ok for people like me, but not sure whether it is ok for others.

@azzamsa maybe we still need to manually write the help page?

Reply:
I change them to lowercase, because #384 had been merged.

> @azzamsa maybe we still need to manually write the help page?

Are those examples #385 sufficient?

Reply:
I prefer to have doc for field `pkg_fmt`, similar to how `Args::log_level` document different options.

Reply:
Would you like to elaborate more? I don't see any much difference here.

```shell
        --log-level <LEVEL>    Utility log level [default: info] [possible values: trace, debug,
                               info, warn, error, off]
                              
```

```shell
        --pkg-fmt <PKG_FMT>
            Override Cargo.toml package manifest pkg-fmt [possible values: tar, tbz2, tgz, txz,
            tzstd, zip, bin] 
 ```

Reply:
If you run `cargo binstall --help`, you would see that the help page of `--log-level` contains brief explanation for each possible value, something like this

```
tbz2: Download format is TAR + Bzip2.
tgz: Download format is TGZ (TAR + GZip).
```

Reply:
Is this looks okay?

```
        --pkg-fmt <PKG_FMT>
            Override Cargo.toml package manifest pkg-fmt.

            The availiable package formats are:

            - tar: download format is TAR (uncompressed)

            - tbz2: Download format is TAR + Bzip2

            - tgz: Download format is TGZ (TAR + GZip)

            - txz: Download format is TAR + XZ

            - tzstd: Download format is TAR + Zstd

            - zip: Download format is Zip

            - bin: Download format is raw / binary

            [possible values: tar, tbz2, tgz, txz, tzstd, zip, bin]

        --pkg-url <PKG_URL>
            Override Cargo.toml package manifest pkg-url
```